.travis.yml

```diff
@@ -29,6 +31,8 @@ install:
         pip install -U tox ;;
       docs)
         pip install -U sphinx ;;
+      quality)
+        pip install -rrequirements/quality.txt
```

Comment:
Sounds good, I've created https://github.com/OpenIDC/pyoidc/issues/301 to keep track of that then.